pandas/core/arrays/arrow/array.py

```diff
@@ -521,6 +525,44 @@ def dropna(self: ArrowExtensionArrayT) -> ArrowExtensionArrayT:
         else:
             return type(self)(pc.drop_null(self._data))
 
+    @doc(ExtensionArray.fillna)
+    def fillna(
+        self: ArrowExtensionArrayT,
+        value: object | ArrayLike | None = None,
+        method: FillnaOptions | None = None,
+        limit: int | None = None,
+    ) -> ArrowExtensionArrayT:
+
+        value, method = validate_fillna_kwargs(value, method)
+
+        if isinstance(self._data.type, pa.DurationType):
```

Comment:
Could this be put in a try except instead? So if this is implemented in a future pyarrow version, this still won't be going down the `super` path

Reply:
maybe with a TODO with condition upon which it can be removed

Reply:
Updated to use try except along with a future TODO.